packages/react-instantsearch/src/connectors/connectInfiniteHits.js

```diff
@@ -5,6 +5,10 @@ import {PropTypes} from 'react';
 
 import createConnector from '../core/createConnector';
 
+function getId() {
+  return 'pager';
```

Comment:
I didn't think that much on that topic. My thinking here is that those ids are for internal use so I didn't think it would matter...